x-pack/plugins/observability_solution/apm/server/lib/helpers/get_apm_alerts_client.ts

```diff
@@ -31,7 +31,13 @@ export async function getApmAlertsClient({
 
   const ruleRegistryPluginStart = await plugins.ruleRegistry.start();
   const alertsClient = await ruleRegistryPluginStart.getRacClientWithRequest(request);
-  const apmAlertsIndices = await alertsClient.getAuthorizedAlertsIndices(['apm']);
+  const apmAlertsIndices = await alertsClient.getAuthorizedAlertsIndices([
+    'logs',
```

Comment:
Are you referring to something shared between the Services Inventory and Service Detail Overview? Or between the Entity Inventory and APM? Sorry, it's not entirely clear, as in this PR we only have this index specification in this file.

Reply:
ah sorry. I messed up with "inventory". I meant "Service inventory"

Reply:
I totally agree with the point that "keeping them separate makes it easy to forget to update one of them," but the context and values used in both changed files are different.

- **/apm/server/lib/helpers/get_apm_alerts_client.ts**: `getAuthorizedAlertsIndices` accepts any `string` as `featureIds`.
- **/apm/public/components/app/alerts_overview/index.tsx**: `AlertsStateTable` uses the `ValidFeatureId` type for `featureIds`, where `ValidFeatureId` = `AlertConsumers`, so we are using its values.

I don't think it's a good idea to use `AlertConsumers` values in `getAuthorizedAlertsIndices`, the contexts are different. Additionally, if we consider doing it the other way around, it's not safe to cast plain strings to `ValidFeatureId`/`AlertConsumers`, as it could lead to invalid values.

Do you have any other suggestions that I might not be seeing clearly?

Reply:
I'm having the same discussion with myself for `Infra`...
I think having everything at the same place can be better, casting `ValidFeatureId` to a string is not a problem, and the other way around it shouldn't happen as we will create the const using `AlertConsumers`.

So making that const in a shared/common folder that both client code and server code can access is the best approach IMO

Reply:
Created in `apmAlertingFeatureIds` https://github.com/elastic/kibana/pull/202025/commits/b592037eda726c51e0e9056e78a063d735e0ace5 👍🏻